app/models/concerns/aceable.rb

```diff
@@ -3,5 +3,9 @@ module Aceable
 
   included do
     has_many :access_control_entries, :as => :aceable
+
+    def self.aceable?
```

Comment:
The method name `aceable?` feels confusing to me, it is not a common term that I would expect someone looking at the code to quickly identify and therefore makes the code harder to read.

Suggest `supports_access_control_entries?` or just `supports_access_control?`